### PR TITLE
update(falcosidekick): Upgrade to v2.28

### PR DESCRIPTION
**Pull request by Lowaiz**
**What type of PR is this?**

 /kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

/kind chart-release

**Any specific area of the project related to this PR?**

/area falcosidekick-chart

**What this PR does / why we need it**:

Update values, env and secrets to fit new features of the next falcosidekick update

**Which issue(s) this PR fixes**:

None

**Special notes for your reviewer**:

Ping to @Issif, with the large milestone that v2.28 is, I think that might help a little bit. 

**Checklist**

- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

**Comment by Issif** [comment]
Can you update this PR with this last feature, I'll review then https://github.com/falcosecurity/falcosidekick/pull/565

Thanks

**Comment by Lowaiz** [comment]
Updated to the last merged PR.
Ping @Issif

File: falcosidekick/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,28 @@ numbering uses [semantic versioning](http://semver.org).
...
+* Add Drop Event config to `alertmanager` output
+* Add `customheaders` to `elasticsearch` output
+* Add `customheaders` to `loki` output
+* Add `customheaders` to `grafana` output
+* Add `externalid` to `aws` output
+* Add `tls` to `smtp` output
+* Add `method` to `webhook` output
+* Add `customattributes` to `gcp.pubsub` output
+* Add `region` to `pargerduty` output
+* Add `topiccreation` to `kafka` output
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:23]:
```suggestion
* Add `topiccreation` and `tls` to `kafka` output
```

File: falcosidekick/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,28 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.6.3
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:8]: This PR is missing https://github.com/falcosecurity/falcosidekick/pull/421/files#

File: falcosidekick/templates/deployment.yaml [hunk 3]
```diff
@@ -84,8 +84,31 @@ spec:
...
+            - name: MUTUALTLSCLIENT_CACERTFILE
+              value: {{ .Values.config.mutualtlsclient.cacertfile | quote }}
+            - name: TLSSERVER_DEPLOY
+              value: {{ .Values.config.tlsserver.deploy | quote }}
+            - name: TLSSERVER_CERTFILE
+              value: {{ .Values.config.tlsserver.certfile | quote }}
+            - name: TLSSERVER_KEYFILE
+              value: {{ .Values.config.tlsserver.keyfile | quote }}
+            - name: TLSSERVER_CACERTFILE
+              value: {{ .Values.config.tlsserver.cacertfile | quote }}
```
Comment (Issif) [inline falcosidekick/templates/deployment.yaml:104]: We need to go further, as nothing is already there to add the cert and key files, these settings are useless. We need to mount secrets as volumes.

File: falcosidekick/values.yaml [hunk 4]
```diff
@@ -57,10 +57,34 @@ config:
...
+    # -- server certification file for TLS Server
+    certfile: "/etc/certs/server/server.crt"
+    # -- server key file for TLS Server
+    keyfile: "/etc/certs/server/server.key"
+    # -- if true mutual TLS server will be deployed instead of TLS, deploy also has to be true
+    mutualtls: false
+    # --  CA certification file for client certification if mutualtls is true
+    cacertfile: "/etc/certs/server/ca.crt"
+    # -- port to serve http server serving selected endpoints
+    notlsport: 2810
```
Comment (Issif) [inline falcosidekick/values.yaml:85]: we need to open this port in the service and ingress with the excluded paths from TLS

**Review by Issif — CHANGES_REQUESTED** [review]

File: falcosidekick/templates/deployment.yaml (hunk 3 above)
Comment (Lowaiz) [reply falcosidekick/templates/deployment.yaml:104]: Does including TLS and mTLS server configs kinda a repetition of the ingress job ?

With the way of doing thing inside Kubernetes, do we need to let the possibility to activate this kind of feature inside the pod ?

Comment (Lowaiz) [reply falcosidekick/templates/deployment.yaml:104]: I think the best method to handle the TLS and mTLS use cases, for the chart, is to use the Ingress options and not let these 2 append in the pod.

Comment (Issif) [reply falcosidekick/templates/deployment.yaml:104]: Some users expect the pods to communicate with TLS, but OK. Let's keep as is for now.

**Review by Issif — APPROVED** [review]

**Pull request merged**